jupyterhub/Dockerfile

```
@@ -1,63 +1,65 @@
-ARG BASE_IMAGE=ubuntu:18.04
+# syntax=docker/dockerfile:1
+ARG BASE_IMAGE=nvidia/cuda:11.7.1-cudnn8-devel-ubuntu22.04
 # paskino/jupyter   datascience-notebook-cuda11
 FROM ${BASE_IMAGE} as base
```

```suggestion
ARG BASE_CONTAINER=nvidia/cuda:11.7.1-cudnn8-devel-ubuntu22.04
FROM ${BASE_IMAGE} as base
```

just using `docker-stacks` convention